### ISSUE TITLE
docker run error 

### ISSUE THREAD

**Issue by shibd** [jepsen-io/jepsen#462]
## Question

```
➜  docker git:(master) ./up.sh --dev
~/workspace/jworkspace/jepsen/docker ~/workspace/jworkspace/jepsen/docker
-e \e[104m\e[97m[INFO]\e[49m\e[39m JEPSEN_ROOT is not set, defaulting to: /Users/baozi/workspace/jworkspace/jepsen
-e \e[104m\e[97m[INFO]\e[49m\e[39m Running docker-compose with dev config
-e \e[104m\e[97m[INFO]\e[49m\e[39m No need to generate key pair
-e \e[104m\e[97m[INFO]\e[49m\e[39m Copying .. to control/jepsen
tar: Option --exclude-ignore=.gitignore is not supported
Usage:
  List:    tar -tf <archive-filename>
  Extract: tar -xf <archive-filename>
  Create:  tar -cf <archive-filename> [filenames...]
  Help:    tar --help
```

##  System Version
- macos: 10.15.4
- docker version: 19.03.5

**Comment by stevana** [comment]
Can you check if the fix in PR #448 helps?

**Comment by shibd** [comment]
It solves the problem
